buildScript/plugin/shadowquic/init.sh

```
@@ -0,0 +1,26 @@
+export RUST_ANDROID_GRADLE_LINKER_WRAPPER_PY=$SRC_ROOT/buildScript/rust-linker/linker-wrapper.py
+export RUST_ANDROID_GRADLE_CC_LINK_ARG=""
+export BINDGEN_EXTRA_CLANG_ARGS=--sysroot=$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/sysroot/
+
+CURR="plugin/shadowquic"
+CURR_PATH="$SRC_ROOT/$CURR"
+
+ROOT="$CURR_PATH/src/main/jniLibs"
+OUTPUT="shadowquic"
+LIB_OUTPUT="lib$OUTPUT.so"
+
+cd $CURR_PATH/src/main/rust/shadowquic
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)